WordPress/WordPressTest/KeychainUtilsTests.swift

```diff
@@ -165,4 +60,27 @@ final private class SFHFKeychainUtilsMock: SFHFKeychainUtils {
         let group = accessGroup ?? "default"
         keychain[group]?[serviceName]?.removeValue(forKey: username)
     }
+
+    override class func getAllPasswords(forAccessGroup accessGroup: String!) throws -> [[String: String]] {
```

Comment:
Is force unwrap because it is overriding an obj-c func?

Reply:
Yeah, that's an Obj-C function I'm overriding. The other overrides in this mock are similar to this one.